.eslintrc.json

```diff
@@ -3,8 +3,11 @@
   "settings": {
     "react": { "version": "16" }
   },
+  "globals": {
```

Comment:
uwu

Reply:
`__DEV__` is global in React Native, so I see no reasons not to include it here

Reply:
https://www.internetslang.com/UWU-meaning-definition.asp